lib/imports/api/authority/server/authority-manager.js

```diff
@@ -52,7 +53,8 @@ export class AuthorityManager {
       // allow clients to request latest doc state
       'ProseMeteor.latestDocState' ({ docId }) {
         check(docId, String);
-        return self.latestDocState({ docId });
+        const connection = this.connection;
```

Comment:
when a user calls `ProseMeteor.latestDocState` I store the DDP connection, so that we can listen for its `onClose` later on